compilers/concrete-compiler/compiler/src/main.cpp

```diff
@@ -508,7 +508,6 @@ cmdlineCompilationOptions() {
       cmdline::optimizerMultiParamStrategy;
   options.optimizerConfig.encoding = cmdline::optimizerEncoding;
   options.optimizerConfig.cache_on_disk = !cmdline::optimizerNoCacheOnDisk;
-  options.optimizerConfig.composable = cmdline::optimizerAllowComposition;
```

Comment:
Also remove composable from command line, as we said we will later give the "composable set" in the MLIR.